### PR TITLE
There and back again

### PR DESCRIPTION
**Pull request by Arkham**
🍨

### CODE REVIEW

File: content/blog/there-and-back-again/index.md [hunk 1]
```diff
@@ -0,0 +1,505 @@
...
+        _ ->
+            Nothing
+```
+
+In this [Ellie app](https://ellie-app.com/cDqjV9LLFX6a1), we can see
+that everything is working nicely :)
+
+## An interesting pair
+
+We notice a terrible bug in our application: we forgot about the Pink
```
Comment (juanedi) [inline content/blog/there-and-back-again/index.md:203]: HA HA HA! I totally fell for that and thought you had actually forgotten about Pink 😆

File: content/blog/there-and-back-again/index.md [hunk 2]
```diff
@@ -0,0 +1,505 @@
...
+
+        "Blue" ->
+            Just Blue
+
+        _ ->
+            Nothing
+```
+
+In this [Ellie app](https://ellie-app.com/cDqjV9LLFX6a1), we can see
+that everything is working nicely :)
```
Comment (juanedi) [inline content/blog/there-and-back-again/index.md:199]: FWIW, I think everything up to this point could be condensed a bit. My thinking:
  - Until now, the text was fun to follow but the motivation wasn't clear to me
  - I'm guessing that anyone that's likely to appreciate the problem and try out elm-review to work around that will probably have seen many basic TEA examples already and doesn't need that much hand holding

File: content/blog/there-and-back-again/index.md [hunk 3]
```diff
@@ -0,0 +1,505 @@
...
+```
+
+Now when we add a new variant of `Suit`, the Elm compiler will nudge us to
+update `toString`, while elm-review will nudge us to add the missing
+variant to `allSuits`, which in turn is going to make sure that `toString`
+and `fromString` will never go out of sync. Here's a link to the [final
+version](https://ellie-app.com/cDrHpYTXsr9a1) of the code for you to
+peruse.
+
+Thanks for reading! 👋
```
Comment (juanedi) [inline content/blog/there-and-back-again/index.md:505]: Probably out of scope of this post, but this got me curious into how the rule works and how I would go about writing my own rules.

Are you planning to write about that? If you are, adding a plug for that next post at the end might be nice.

**Review by juanedi — COMMENTED** [review]
Niiiiiiice! Left some minor comments :-D

File: content/blog/there-and-back-again/index.md (hunk 3 above)
Comment (Arkham) [reply content/blog/there-and-back-again/index.md:505]: Yes! next post incoming!

File: content/blog/there-and-back-again/index.md (hunk 2 above)
Comment (Arkham) [reply content/blog/there-and-back-again/index.md:199]: Yup, good point, I've thought about doing that before.

**Comment by Arkham** [comment]
Thank you again @juanedi  ❤️

**Pull request merged**